lib/ubygems.rb

```diff
@@ -7,5 +7,8 @@
 # See LICENSE.txt for permissions.
 #++
 
+# TODO: Remove this file at RubyGems 4.0
+# Based by https://bugs.ruby-lang.org/issues/14322#note-4
+warn "`ubygems.rb' is deprecated, It will be removed on or after 2018-12-01. remove `-rubygems' from your command-line"
```

Comment:
@hsbt what do you think of this message?:

```
warn "`ubygems.rb' is deprecated, and will be removed on or after 2018-12-01. Remove `-rubygems' from your command-line, or use `-r rubygems' instead"
```

Reply:
👍